### PR TITLE
Maui ebook preface

### PR DESCRIPTION
**Pull request by michaelstonis**
## Summary

Provides the preface chapter to the MAUI ebook.

### CODE REVIEW

File: docs/architecture/maui/preface.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+title: A preface to enterprise architecture for MAUI applications
+description: Guidance for what the book will cover, reference materials, and how to approach the book
+author: michaelstonis
+no-loc: [MAUI]
+ms.date: 06/18/2022
+---
+# Introduction
+
+## Purpose
+
```
Comment (IEvangelist) [inline docs/architecture/maui/preface.md:11]: It's not ideal to have one heading followed immediately by a sub-heading. I suggest getting rid of the Purpose subheading, as this is part of the introduction.
```suggestion
```

File: docs/architecture/maui/preface.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: A preface to enterprise architecture for MAUI applications
+description: Guidance for what the book will cover, reference materials, and how to approach the book
```
Comment (IEvangelist) [inline docs/architecture/maui/preface.md:3]:
```suggestion
description: Guidance for what the book will cover, reference materials, and how to approach the book.
```

Missing punctuation.

**Review by IEvangelist — APPROVED** [review]
Two minor suggestions that I'd like to see before merging.

**Comment by IEvangelist** [comment]
Also, please address the two MD linter warnings/errors:

- Error: docs/architecture/maui/index.md:71 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
- Error: docs/architecture/maui/index.md:81:24 MD047/single-trailing-newline Files should end with a single newline character

File: docs/architecture/maui/preface.md (hunk 1 above)
Comment (michaelstonis) [reply docs/architecture/maui/preface.md:11]: Great catch. After flipping it back and forth a few times, I think keeping the purpose section reads better.

**Pull request merged**